account_banking_payment_export/models/payment_mode.py

```diff
@@ -43,9 +43,14 @@ def suitable_bank_types(self, cr, uid, payment_mode_id=None, context=None):
             res = [t.code for t in payment_mode.type.suitable_bank_types]
         return res
 
+    def _default_type(self):
+        return self.env.ref('account_banking_payment_export.'
+                            'manual_bank_tranfer')
+
     type = fields.Many2one(
```

Comment:
@sbidoul I'll do the changes 
Unfortunately, the @api.returns('payment.mode.type') is without effect in this case (as I remember) (a odoo bug?)